lumber_yard.rb

```diff
@@ -86,31 +91,43 @@
     username: params[:username],
     employee_type: params[:employee_type],
     }).valid?
-    @options = ModelCitizen::Messages.new
-    @success = false
-    @message = ModelCitizen::Messages.new.get_message(:invalid_employee)
-    erb :'employee/form'
+    flash[:employee_error] = ModelCitizen::Messages.new.get_message(:invalid_employee)
+    redirect '/employees/new'
   else
-    @options = ModelCitizen::Messages.new
-    @success = true
-    @message = ModelCitizen::Messages.new.get_message(:employee_success)
-    erb :'employee/form'
+    flash[:employee_success] = ModelCitizen::Messages.new.get_message(:employee_success)
+    redirect '/home/index'
   end
 end
 
-post '/clients/create' do
+post '/clients' do
   if !LumberYard::Client.new.create_client({
     name: params[:name],
     type: params[:type]
     }).valid?
-    @options = ModelCitizen::Messages.new
-    @success = false
-    @message = ModelCitizen::Messages.new.get_message(:invalid_client)
-    erb :'client/form'
+    flash[:client_error] = ModelCitizen::Messages.new.get_message(:invalid_client)
+    redirect '/clients/new'
   else
-    @options = ModelCitizen::Messages.new
-    @success = true
-    @message = ModelCitizen::Messages.new.get_message(:client_success)
-    erb :'client/form'
+    flash[:client_success] = ModelCitizen::Messages.new.get_message(:client_success)
+    redirect '/home/index'
   end
+end
+
+def set_session_data(employee)
```

Comment:
Re: my other comment about getters and setters in Java vs Ruby - this reads OK here, but I think that because you are doing such simple things inside of `set_username_session`, `set_first_name_session`, and `set_type_session`, you should just inline those methods back into `set_session_data`